src/templates/en/index.html

```diff
@@ -69,11 +70,50 @@ <h2>
         </button>
       </div>
       <div class="intro-year">2019</div>
-      <div class="intro-image">
-        <img class="home-browser" src="/static/images/home-browser.png" />
-        <img class="camada-751" src="/static/images/camada-751.png" />
-        <img class="camada-759" src="/static/images/camada-759.png" />
-        <img class="camada-752" src="/static/images/camada-752.png" />
+      <div class="intro-image-container">
+        <div class="intro-image">
+          <img class="home-browser" src="/static/images/home-browser.png" />
+          <img class="camada-751" src="/static/images/camada-751.png" />
+          <img class="camada-759" src="/static/images/camada-759.png" />
+          <img class="camada-752" src="/static/images/camada-752.png" />
+        </div>
+      </div>
+      <div class="menu">
```

Comment:
Any reason why the menu isn't part of the nav section, adjacent to the menu icons?

Reply:
![image](https://user-images.githubusercontent.com/25320782/65422297-9fad9980-de38-11e9-86d7-6b629db8a528.png)

The menu now appears below the nav bar and placed on top of the `intro-container` section.